rllib/examples/env/coin_game_non_vectorized_env.py

```diff
@@ -46,14 +52,13 @@ def __init__(self, config: Optional[Dict] = None):
         self._load_config(config)
         self.player_red_id, self.player_blue_id = self.players_ids
         self.n_features = self.grid_size**2 * (2 * self.NUM_AGENTS)
-        self.OBSERVATION_SPACE = gym.spaces.Box(
+        self.observation_space = gym.spaces.Box(
             low=0, high=1, shape=(self.grid_size, self.grid_size, 4), dtype="uint8"
         )
 
         self.step_count_in_current_episode = None
         if self.output_additional_info:
             self._init_info()
-        self.seed(seed=config.get("seed", None))
```

Comment:
Neither essential to this example, does self.seed exist anymore.

Reply:
ok so to be clear, this is the broken code and we deleted it, but can users set the seed via a call to reset now?

Reply:
Yes, the reset method calls from gymnasium.utils.seeding with the user provided seed!

Reply:
mk :)